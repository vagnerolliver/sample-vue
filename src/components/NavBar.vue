<template>
  <nav class="navbar" role="navigation" aria-label="main navigation">
    <a class="navbar-item">
        My Page 1
    </a>
    <a class="navbar-item">
        My Page 2
    </a>
  </nav>
</template>

<script>
  export default {
    name: 'NavBar',
  }
</script>


