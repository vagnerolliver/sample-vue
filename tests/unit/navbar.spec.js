import { mount } from '@vue/test-utils'
import NavBar from '@/components/NavBar'


describe('Component', () => {
  test('is a Vue instance', () => {
    const wrapper = mount(NavBar)
    expect(wrapper.isVueInstance()).toBeTruthy()
  })

  test('snapshot renders correctly', () => {
    const wrapper = mount(NavBar)
    expect(wrapper.element).toMatchSnapshot()
  })
})
