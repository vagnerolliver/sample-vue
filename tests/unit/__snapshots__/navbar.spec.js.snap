// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component snapshot renders correctly 1`] = `
<nav
  aria-label="main navigation"
  class="navbar"
  role="navigation"
>
  <a
    class="navbar-item"
  >
    
      My Page 1
  
  </a>
   
  <a
    class="navbar-item"
  >
    
      My Page 3
  
  </a>
</nav>
`;
